modules/perception/inference/libtorch/torch_net_test.cc

```diff
@@ -0,0 +1,86 @@
+/******************************************************************************
+ * Copyright 2020 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include <torch/torch.h>
+#include <torch/script.h>
+#include <opencv2/opencv.hpp>
+
+#include "cyber/common/file.h"
+#include "modules/perception/base/blob.h"
+#include "modules/perception/camera/common/util.h"
+#include "modules/perception/inference/inference.h"
+#include "modules/perception/inference/inference_factory.h"
+#include "modules/perception/inference/utils/resize.h"
+
+int main() {
+  std::shared_ptr<apollo::perception::inference::Inference> rt_net_ = nullptr;
```

Comment:
local variables in free functions should not be ended with "_".  "var_" were for class member variables.